registry/server/index.ts

```diff
@@ -1,11 +1,9 @@
-import path from 'path';
+import 'newrelic';
+
 import app from './app';
 import server from './server';
 import { getLogger } from './util/logger';
 
-process.env.NODE_CONFIG_DIR = path.resolve(__dirname, '../config');
```

Comment:
are you sure that this line is not needed? It seems that it resolves to a correct newrelic config file